Untitled.py

```diff
@@ -0,0 +1,53 @@
+
+
+
+def sheet_data(sheet):
+
+    out = list()
+
+    sheetIndex = 0
+    for X in range(1, s.nrows):
```

Comment:
1)low  letters are common

Reply:
It's pretty hard to guess the problem from the description. What exactly do you mean?

Reply:
 Sorry! :) During the lesson, you mentioned that usually, people use lowercase for variables names